siriuspy/siriuspy/devices/energy.py

```diff
@@ -0,0 +1,106 @@
+"""Energy Devices."""
+
+from .device import DeviceApp as _DeviceApp
+
+
+class Energy(_DeviceApp):
+    """Energy devices."""
+
+    class DEVICES:
+        """."""
+
+        TB = 'TB'
```

Comment:
@xresende, I think it is better to use `namedtuple` here.

Reply:
@fernandohds564 , I was waiting for you to submit a PR with your namedtuple implementation in order to convert in all device classes. By looking at your example (same v variable names, structures, etc), we could enforce standardization.

Reply:
ok! I think it will take sometime for that branch of mine become PRable.
Let's follow your strategy and standardize later.